moz/classified-flood-hazard-zone/flood-hazard-zone.json

```diff
@@ -25,5 +25,5 @@
     "popup": {
       "body": "<p style=text-align:left;>Rio/River: {{rio}} <br/>{{Level}}</p>"
     },
-    "credit": "Source: <a href='https://geonode.ingc.gov.mz/layers/geonode:national_flood_multi' target='_blank'>National Directorate of Water Resource Management (DNGRH)</a>"
+    "credit": "Source: https://geonode.ingc.gov.mz/layers/geonode:national_flood_multi"
```

Comment:
@KipSigei let's do `Geonode - national_flood_multi` for this (the title tag will have `Credit: ` prepended and let's just do the ID of the node instead of the whole URL